package.json

```diff
@@ -18,21 +18,34 @@
   },
   "dependencies": {
     "@erc725/erc725.js": "0.16.0",
-    "@lukso/lsp-factory.js": "2.4.0",
-    "@lukso/lsp-smart-contracts": "0.7.0",
+    "@lukso/lsp-factory.js": "^2.4.0",
+    "@lukso/lsp-smart-contracts": "^0.7.0",
     "@walletconnect/client": "1.8.0",
+    "@walletconnect/encoding": "^1.0.1",
+    "@walletconnect/ethereum-provider": "^2.4.4",
     "@walletconnect/qrcode-modal": "1.8.0",
+    "@walletconnect/sign-client": "2.2.1",
+    "@walletconnect/types": "2.2.1",
+    "@walletconnect/utils": "2.2.1",
     "@walletconnect/web3-provider": "1.8.0",
+    "@web3modal/ethereum": "^2.1.1",
+    "@web3modal/standalone": "^2.1.1",
     "bulma": "0.9.4",
     "eth-rpc-errors": "4.0.3",
+    "ethers": "^5.7.2",
     "filesize": "10.0.6",
     "https-browserify": "1.0.0",
     "siwe": "1.1.6",
     "vue": "^3.2.45",
     "vue-router": "4.1.6",
+    "wagmi": "^0.11.4",
     "web3": "1.8.1",
     "web3-eth-contract": "^1.8.1"
   },
+  "resolutions": {
+    "@lukso/lsp-factory.js/ethers": "^5.7.0",
+    "uint8arrays": "^3.1.1"
```

Comment:
@richtera Thanks again for this trick! Resolved an issue with 3rd party dependency and won't have to wait for it to be updated: 
https://github.com/WalletConnect/walletconnect-monorepo/issues/1329#issuecomment-1438182513
https://github.com/WalletConnect/walletconnect-monorepo/pull/2024